vertical-pod-autoscaler/pkg/updater/logic/updater_test.go

```diff
@@ -209,11 +177,14 @@ func TestRunOnce_InvalidStatus(t *testing.T) {
 		recommendationProcessor:      &test.FakeRecommendationProcessor{},
 		selectorFetcher:              mockSelectorFetcher,
 		useAdmissionControllerStatus: true,
-		statusValidator:              newFakeValidator(false),
+		statusValidator:              statusValidator,
 	}
 
+	if expectFetchCalls {
```

Comment:
Is there value from verifying those calls are not made if not expected?

Reply:
1. This verification was in original tests, so I want to preserve the same logic.
2. IIUC using gomock controller defines a list of calls that we expect to happen. If the call is registered and not used it throws exception at the end of the test.
This is the first time I'm seeing this mock controller so I can be wrong here...

Reply:
I see, thanks